### PR TITLE
Stanzas are sorted by `Term` and `Typedef`

### PR DESCRIPTION
**Pull request by hrshdhgd**

### CODE REVIEW

File: src/oaklib/datamodels/vocabulary.py [hunk 1]
```diff
@@ -146,6 +146,9 @@
 OWL_VERSION_INFO = "owl:versionInfo"
 OWL_VERSION_IRI = "owl:versionIRI"
 
+OBO_TERM = "Term"
```
Comment (cmungall) [inline src/oaklib/datamodels/vocabulary.py:149]: these shouldn't go in vocabulary as it's entirely an obo syntax thing - aren't these already in simple obo parser?

File: src/oaklib/implementations/simpleobo/simple_obo_parser.py [hunk 2]
```diff
@@ -596,9 +600,25 @@ def order_stanzas(self) -> None:
 
         Does not change tag-value ordering within stanzas
         """
-        stanzas = self.stanzas.values()
-        stanzas = sorted(stanzas, key=lambda x: x.id)
-        self.stanzas = {s.id: s for s in stanzas}
+        term_stanzas = {
```
Comment (cmungall) [inline src/oaklib/implementations/simpleobo/simple_obo_parser.py:603]: there may be more compact ways to write this, but the logic is perfect I think!

**Review by cmungall — CHANGES_REQUESTED** [review]
just the vocabulary change

**Comment by hrshdhgd** [comment]
Done!

**Review by cmungall — APPROVED** [review]

**Pull request merged**